README.md

```diff
@@ -208,94 +181,134 @@ type RegisterProps = {
 - The initial Register page has two input text boxes "First Name" and "last Name". The next button uses the toggle function to conditionally render an alternative register page for the second half of the account creation process. On the initial page the back arrow navigates back to the splash page.
 
 - The secondary register page has three input boxes. Email, Password and Confirm Password. The create account button checks the password and email eligibility and then if correct submits that information to the fireBase. The Back button on this page sends the user back to the first Register page.
-  <<<<<<< HEAD
-
-### Events Page
-
-- Created a filter that ensures the only events visible are events in the future from today's date.
-- Created events page to render the search bar and events passed in.
-- The search bar filters the events by event name, category and description
-- State for each events' "booking" button is kept in Events as an array
-
-```tsx
-const [buttonVariants, setButtonVariants] = useState<boolean[]>(
-  new Array(eventData.length).fill(false)
-);
-```
 
-- Use index when mapping over events to target the buttons individually
+### Error
 
-```tsx
-{filteredSearch.map((event: MockEvent, index: number))}
-```
+- **Error Message**: When a user lands on a non-existent route or page, they are presented with an error message indicating that the requested content could not be found.
+- **Return to Home Button**: To enhance user navigation and convenience, a button is provided on the 404 Not Found Page. When clicked, this button redirects the user back to the home page.
 
-- Added feature whereby users are able to see how many spaces are available dynamically through the database.
+### Admin
 
-### Calendar
+- Describe the Admin page if applicable.
```

Comment:
I think we can give a short overview of this page now.

Reply:
added chu and jugz admin page notes